ports/projectm4/portfile.cmake

```
@@ -0,0 +1,35 @@
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO projectM-visualizer/projectm
+    REF "f60cd86"

```

```suggestion
    REF f60cd86ab687e7b65e6e36a8f0a606dd1141ea23
```
It might be desirable to switch the regular `version` scheme in the future, which would allow to express git tags like `v${VERSION}`.

---

Thanks @dg0yt .
@Blaquewithaq Could you help switch to the release version like https://github.com/projectM-visualizer/projectm/releases/tag/v4.1.1?

---

> Could you help switch to the release version like https://github.com/projectM-visualizer/projectm/releases/tag/v4.1.1?

This was already discussed aboved, we had to make a small change in the build system in response to the above discussion, and this commit is not in the 4.1.1 release. So to resolve this, we have basically three options:
1. We can revert to the 4.1.1 version and have to use the submodule option for the `projectm-eval` dependency, which, as stated, was requested to be changed to an external dependency.
2. Use the commit hash as-is in the port, which, apart form some cosmetic and GitHub-related changes, is essentially the same as v4.1.1
3. Wait until the next projectM version is released which includes the projectm-eval dependency fix for vcpkg. In this case, we should close/draft this PR, as this might take a few onths like Blaque said.

I'd personally be fine with keeping the commit hash, even if it's not perfect, and when we've released the next version (either 4.1.2 or 4.2.0), we'd update the port and go back to using the variable-based version tag instead.

---

> I'd personally be fine with keeping the commit hash, even if it's not perfect, and when we've released the next version (either 4.1.2 or 4.2.0), we'd update the port and go back to using the variable-based version tag instead.

IMO that's fine for the first version of the new port.
And my suggested change is to use the full SHA1 now, not to switch versioning now.